qemu/tests/qemu_img_bitmap.py

```diff
@@ -69,6 +74,11 @@ def _check_bitmap_add(img, bitmap_name):
     bitmap_img = QemuImg(params.object_params(bitmap), root_dir, bitmap)
     bitmap_name = params["bitmap_name"]
 
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    match = re.search(r'[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?', qemu_version)
+    host_qemu = match.group(0)
+
```

Comment:
Never use a 'private' function, which may be changed in VT while tp-qemu cannot be aware of the change.

qemu_binary = utils_misc.get_qemu_binary(self.params)
qemu_version = utils_qemu.get_qemu_version(qemu_binary)[0]
if qemu_version in VersionInterval(...):